### PR TITLE
Set up Space Travelers' Hub Website

### PR DESCRIPTION
**Pull request by ShadyShawkat**
# We worked on the following tasks/features:

- Add React Components ( Navbar - Missions - Rockets - Profile )
- React Props
- React Routers
- Redux ToolKit
- Handle events with React
- Style with Tailwind
- Life Cycle
- Best Practices 
- Units tests using Jest
- Store, Actions, and reducer
- We made use of the React documentation
- Performed code reviews

### CODE REVIEW

File: src/components/Navbar/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+      <li>
+        <NavLink to="/">Rockets</NavLink>
+      </li>
+      <li className="border-l-black border-l-2 pl-3">
+        <NavLink to="missions">Missions</NavLink>
+      </li>
+      <li className="border-l-black border-l-2 pl-3">
+        <NavLink to="profile">Profile</NavLink>
+      </li>
+    </ul>
```
Comment (ZeenatLawal) [inline src/components/Navbar/NavBar.js:21]: - [x] The active page should be indicated with an underline as stated in the requirement `style active class to indicate which section/page user is currently on (underline active navigation link)`. Kindly add the styling.

File: src/components/Missions/Missions.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```
Comment (ZeenatLawal) [inline src/components/Missions/Missions.js:1]: - [x] Kindly write more unit tests for the other components using React Testing Library and Jest.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @ShadyShawkat and @iwillteixeira, 

Good job so far!:clap: :clap:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎊 🎊 
- Git flow is followed. ✔️
- PR has a title and summary. ✔️


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi Team🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**